src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -33,31 +34,35 @@ class VisitService(
   private val visitRepository: VisitRepository,
   private val supportTypeRepository: SupportTypeRepository,
   private val telemetryClient: TelemetryClient,
+  private val snsService: SnsService,
 ) {
 
-  fun createVisit(createVisitRequest: CreateVisitRequestDto): VisitDto {
+  fun reserveVisit(useReference: String = "", reservedVisitRequestDto: ReserveVisitRequestDto): VisitDto {
```

Comment:
should useReference be null instead of "" for DB to populate null?

Reply:
I will look into this forgotten why I have done it this way, may be it's me not being used to kotlin